torchbenchmark/models/pytorch_unet/__init__.py

```diff
@@ -0,0 +1,87 @@
+import argparse
+import numpy as np
+import random
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch import optim
+
+random.seed(1337)
+torch.manual_seed(1337)
+np.random.seed(1337)
+torch.backends.cudnn.deterministic = True
+torch.backends.cudnn.benchmark = False
+
+from .pytorch_unet.unet import UNet
+from .pytorch_unet.utils.dice_score import dice_loss
+
+from ...util.model import BenchmarkModel
+from torchbenchmark.tasks import COMPUTER_VISION
+
+
+class Model(BenchmarkModel):
+
+    task = COMPUTER_VISION.SEGMENTATION
+
+    def __init__(self, device=None, jit=False):
+        super().__init__()
+        self.device = device
+        self.jit = jit
+
+        self.args = self._get_args()
+        self.sample_inputs = torch.rand((1, 3, 640, 959), dtype=torch.float32).to(self.device)
```

Comment:
Is the shape of the sample_inputs here the same as the original data input?

Reply:
Yes, I printed the shape by downloading the actual data referred in the original repo and running train.

Reply:
Add a comment explaining where the shape came from, link to data, etc. Ideally you can describe what the sizes mean. For example is first size the batch_size?

Reply:
Added comments.